### PR TITLE
feat: this add post update after comment

### PR DESCRIPTION
**Pull request by vnko1**

### CODE REVIEW

**Review by ArtemBohak — APPROVED** [review]

File: src/pages/ProfilePage/components/Profile/components/PostSkeleton/PostSkeleton.tsx
Comment (BogAlex4ndr) [inline src/pages/ProfilePage/components/Profile/components/PostSkeleton/PostSkeleton.tsx:1]: Why did you delete this component?

**Review by BogAlex4ndr — APPROVED** [review]
